### PR TITLE
Scala 2.12.1 support

### PR DESCRIPTION
**Pull request by golem131**

### CODE REVIEW

File: project/Build.scala [hunk 1]
```diff
@@ -7,6 +7,7 @@ object ProjectBuild extends Build {
   val postgresqlName = "postgresql-async"
   val mysqlName = "mysql-async"
 
+  scalaVersion := "2.12.0"
```
Comment (xuwei-k) [inline project/Build.scala]: This has no effect.

**Comment by golem131** [comment]
@xuwei-k - Fixed

File: project/Build.scala (hunk 1 above)
Comment (golem131) [inline project/Build.scala]: Fixed

**Comment by kildievr** [comment]
+1

**Comment by linusnorton** [comment]
@mauricio what is the process for getting this merged?

**Comment by mihaisoloi** [comment]
+1

**Comment by golem131** [comment]
I mean this with no snark: @mauricio do you intend to continue to maintain this project? (The lack of recent commits seems to suggest the answer may be "no.") If you do intend to keep it alive, it's getting to be a long time to not have cut a Scala 2.12 build, when there's a simple, working PR.

It's perfectly fine if you don't have the time or interest to continue basic maintenance like this. If that's the case, I respectfully suggest that you put a comment to that effect at the top of the README so people can make an informed decision about whether to use this library.

**Comment by linusnorton** [comment]
Does anyone else have commit access? Maurico has zero activity in December... what if he's just gone?

**Comment by augi** [comment]
Hi, are you aware of that the 2.12 version is not released?

**Comment by kildievr** [comment]
2.12 is released, proof: http://scala-lang.org/news/2.12.0

**Comment by augi** [comment]
I'm of course talking about 2.12 version of this library.

**Pull request merged**